test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
```

Comment:
SLC_COMMAND is usually `slc` or `apic`, think about `$0` in shell (the name of the executable that started this process).